SOP/SOP-mdc.adoc

```diff
@@ -0,0 +1,134 @@
+2019/08/01 09:39:04 oauthproxy.go:238: Cookie settings: name:_oauth_proxy secure(https):true httponly:false expiry:168h0m0s domain:<default> refresh:disabled
+2019/08/01 09:39:04 http.go:56: HTTP: listening on 0.0.0.0:4180
+----
++
+
+.. If alternative logs are found in the above step then save the logs by running `oc logs <service-podname> -c mdc-oauth-proxy > <filename>.log`
+
+=== Warning
+
+==== MobileDeveloperConsolePodCPUHigh
+
+This alert indicates that the Service pod(s) is/are using more CPU than expected.
+
+. Please follow the <<To capture the logs>> procedure in order to capture the required information to send it to its maintainers.
+. Please follow the <<To scale the pod>> procedure in order to try to solve performance issues.
+
+==== MobileDeveloperConsolePodMemoryHigh
+
+This alert indicates that the Service pod(s) is/are using more Memory than expected.
+
+. Please follow the <<To capture the logs>> procedure in order to capture the required information to send it to its maintainers.
+. Please follow the <<To scale the pod>> procedure in order to try to solve performance issues.
+
+
+==== To capture the logs
+
+. Capture a snapshot of the 'Mobile Developer Console Application' Grafana dashboard and track it over time. The metrics can be useful for identifying performance issues over time.
+. Capture the Service logs
+.. Get the pod names by running `oc get pods`
+.. Save the logs by running `oc logs <mdc-podname> > <filename>.log` for any pod
+. Capture CR Status by running `oc describe MobileDeveloperConsole`.
+
+==== To scale the pod
+
+. Scale the Mobile Develper Console pod
+.. Check the number of DESIRED and CURRENT replicas, and make sure that they're the same:
+----
+oc get deploymentconfig --selector='app=mdc'
+NAME      REVISION   DESIRED   CURRENT   TRIGGERED BY
+mdc       3          1         1         config,image(mdc-imagestream:1.1.4),image(mdc-oauth-proxy-imagestream:latest)
+----
+If they're not, this may indicate a problem with the ability to scale up (capacity, missing images, etc), or the ability to scale down (a node where a pod was scheduled might be in trouble etc.).
+.. Scale the replication controller if more pods are desired by running `oc scale deploymentconfig --selector='app=mdc' --replicas=X` where X is the number of desired pods.
+
+== Validate
+
+=== Install
+
+. Switch to the Mobile Developer Console namespace by running `oc project <namespace>`  E.g `oc project mobile-developer-console`.
+. Check if the mobile developer console pod to is present by running `oc get pods | grep mdc` or `oc get pods | grep mobile-developer-console`.
++
+[source,shell]
+----
+NAME                                           READY     STATUS    RESTARTS   AGE
+mdc-58c8877fd8-g6dvr                           1/1       Running   3          9d
+----
+
+NOTE: The operator is responsible for managing and creating all objects required to have the Mobile Developer Console Service deployed on the cluster.
```

Comment:
Hi @laurafitzgerald it is missing steps. 
- Checking the CRs
- Check env variables 
- Check OAuth Proxy Container

etc ...